usl_pipeline/usl_lib/usl_lib/storage/metastore.py

```diff
@@ -266,6 +266,31 @@ def update_scaling_done(
             {"needs_scaling": False, "feature_matrix_path": scaled_feature_matrix_path},
         )
 
+    @classmethod
+    def delete_all_for_study_area(
```

Comment:
I think this might make more since as a static method on StudyArea rather than StudyAreaChunks because it logically operates over a StudyArea.

Reply:
Done

Reply:
This is still on StudyAreaChunk -- did you forget to push some code?

Reply:
Oops, sorry. Now should be done.